pkgs/applications/networking/instant-messengers/whatsapp-for-linux/default.nix

```diff
@@ -14,7 +14,7 @@
 
 stdenv.mkDerivation rec {
   pname = "whatsapp-for-linux";
-  version = "1.3.1";
+  version = "1.6.1";
 
   src = fetchFromGitHub {
```

Comment:
The hash must be updated

Reply:
Thank you so much, this is my first commit, so let me know if i need to do anything else from my side to see the pull request solved

Reply:
@discite  you still do not update the hash. Not sure if you have resolved the comment then by accident.
Beside that, there is also a format suggested for the commit messages. That can be read in the contribution guideline (there is a link for that in the PR template).